docs/gpu-driver/gpu-driver.rst

```diff
@@ -0,0 +1,53 @@
+MIGraphX GPU-driver
+===============
+
+The MIGraphX gpu-driver is used to test the performance of GPU kernels produced in MIGraphX.
+Example usage::
+    
+    gpu-driver kernel_test.json
+
+The input json file describes the gpu kernel and the input data settings.
+Random data is passed into the gpu kernel and the kernel is run a set number of iterations
```

Comment:
What is the number of iterations, please.

I ran the example below. I tried -h option, and gpu-driver crashed.